code/__HELPERS/_lists.dm

```diff
@@ -51,6 +51,8 @@
 ///Returns the list if it's actually a valid list, otherwise will initialize it
 #define SANITIZE_LIST(L) ( islist(L) ? L : list() )
 #define reverseList(L) reverse_range(L.Copy())
+/// inserts the given Key Value pair as a list association in L at the given index.
+#define INSERT_ASSOCIATIVE_KV_PAIR(L, K, V, _index) L.Insert(_index, K); L[K] = V;
```

Comment:
Fucking cursed brother. Is this even used or am I blind

Reply:
it was when i made SSrunechat into a non SStimer subtype :trollface:

Reply:
Pogged

Reply:
im sure someone could use it

Reply:
it'd break after the first insert, since one would just shift all the others over 
s not a good pattern I don't think

Reply:
not useful anyways, removed